client/common/cmdline.h

```diff
@@ -224,6 +224,12 @@ static const COMMAND_LINE_ARGUMENT_A global_cmd_args[] = {
 	{ "kbd-unicode", COMMAND_LINE_VALUE_FLAG, "", NULL, NULL, -1, NULL,
 	  "Send unicode symbols, e.g. use the local keyboard map. ATTENTION: Does not work with every "
 	  "RDP server!" },
+	{ "kerberos-start-time", COMMAND_LINE_VALUE_OPTIONAL, "<start time>", NULL, NULL, -1, NULL,
```

Comment:
Maybe reorganzie this, use something similar to `/kerberos:start-time:xxx,lifetime:yyy, ...`?

Reply:
Makes sense (should also wire them because they're unused right now). I should also wire an armor cache parameter so that we can support windows authentication silos.

Reply:
I've moved all these options to `/kerberos:<option>`